core/indigo-core/molecule/molecule_cdxml_loader.h

```diff
@@ -357,8 +357,8 @@ namespace indigo
             }
             break;
             case ECDXType::CDXINT16: {
-                auto ptr16 = (int32_t*)ptr;
-                result = parseCDXINT16(*ptr16, tag);
+                int16_t val16 = sz == 1 ? *((int8_t*)ptr) : *((int16_t*)ptr); // ChemDraw 8.0 bug fix
```

Comment:
move "1" to constant. Make more clear statement  if sz == ... else ...
rename sz to understandable value